protos/sigstore_trustroot.proto

```diff
@@ -80,7 +81,7 @@ message TrustedRoot {
         // intermediate and/or leaf certificates.
         // The certificates are intended to be used for verifying artifact
         // signatures.
-        repeated CertificateAuthority certificate_authorities = 3;
+        repeated CertificateAuthority certificate_authorities = 3 [(google.api.field_behavior) = REQUIRED];
```

Comment:
This one should be optional too. (I think I mentioned that in the previous comment.)

Reply:
Well, if `CertificateAuthority` is optional, then, I guess, the bundle can't have signatures. What do you put into `Bundle.content` if the deployment misses CA?

Reply:
You can, if the artifact is signed by a private key-pair, then there may be no CA involved.